lib/supa/builder.rb

```diff
@@ -21,6 +21,12 @@ def initialize(subject, representer:, tree:)
       end
     end
 
+    def attributes(names, options = {})
```

Comment:
@Strech It is not possible to do it like this:

![builder_rb_ ___projects_dt_supa](https://cloud.githubusercontent.com/assets/159152/23361850/fd760e5a-fcf2-11e6-9f73-3c78e60aa903.png)

We can do it with kvargs `def attributes(*names, options: {})`